src/gpl/src/placerBase.cpp

```diff
@@ -1002,8 +1002,8 @@ void PlacerBase::initInstsForUnusableSites()
 {
   dbSet<dbRow> rows = db_->getChip()->getBlock()->getRows();
 
-  int siteCountX = (die_.coreUx() - die_.coreLx()) / siteSizeX_;
-  int siteCountY = (die_.coreUy() - die_.coreLy()) / siteSizeY_;
+  int64 siteCountX = (die_.coreUx() - die_.coreLx()) / siteSizeX_;
+  int64 siteCountY = (die_.coreUy() - die_.coreLy()) / siteSizeY_;
```

Comment:
int64_t is preferrable as it is defined in the c++ std and int64 is not and may not be present on all platforms.